test/support/fixtures/figgy_test_fixtures.ex

```diff
@@ -58,8 +87,6 @@ defmodule FiggyTestFixtures do
         }
       })
 
-    :timer.sleep(1)
```

Comment:
We don't need these anymore?

Reply:
I don't think so. It got added in https://github.com/pulibrary/dpul-collections/pull/90 while I was trying to figure out that race condition that turned out to be because we were querying the wrong field.